src/main/java/kr/codesqaud/cafe/domain/Article.java

```diff
@@ -0,0 +1,41 @@
+package kr.codesqaud.cafe.domain;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class Article {
+    private final Long id;
+    private final String writer;
+    private final String title;
+    private final String contents;
+    private LocalDate createdDate;
+    private static Long sequence = 0L;
+
+    public String getWriter() {
+        return writer;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public String getContents() {
+        return contents;
+    }
+
+    public Article(String writer, String title, String contents) {
+        this.id = generateId();
+        this.writer = writer;
+        this.title = title;
+        this.contents = contents;
+        this.createdDate = LocalDate.now();
+    }
+
+    public boolean isIdEquals(Long articleId) {
+        return this.id.equals(articleId);
+    }
+
+    private static Long generateId() {
+        return ++ sequence;
```

Comment:
```suggestion
        return ++sequence;
```
간단하게 표현한 건 좋지만 전위연산자는 잘 사용하지 않는 경향이 있습니다.

Reply:
sequence 를 사용하는 코드가 스레드 safe 한지도 고민해 보시면 좋을 것 같아요.